CMakeLists.txt

```
@@ -171,6 +194,13 @@ if(ENABLE_PLASMA)
     find_package(SLATE REQUIRED)
 endif()
 
+# check for preCICE
+if(ENABLE_PRECICE)
+    find_package(precice REQUIRED
```

It may be a good idea to restrict the version, given that preCICE 3 is on the horizon.

```suggestion
    find_package(precice 2 REQUIRED
```